### ISSUE TITLE
[BUG] Address Check feature makes checkout slow in Magento 2.4.1

### ISSUE THREAD

**Issue by ArjenMiedema** [tig-nl/postnl-magento2#218]
**To Reproduce**
1. Ensure you have PostNL enabled and also the Address Check feature is set to "Enabled". Also make sure you have several payment methods enabled
2. Go to the webshop
2. Add any product to the cart
3. Go to the checkout and enter any address
4. When using a test account, the address check will fail
5. Select a shipping method and continue to step 2
6. Select a payment method and check the result

**Expected result**
The payment method is selected and I can place an order

**Actual result**
Selecting the payment method freezes the website for some time before loading the payment method in the screen. In that time you can move your mouse, but can't/do click anything in the website. Also the loader is not showing up until the end of the "timeout".

**Workaround**
Disabling the address check feature solves this issue. It's also possible to remove the foreach loops in `Tig_Postnl/view/frontend/web/template/billing-address/details.html`. The foreach loops `<!-- ko foreach: { data: Object.keys(element), as: 'attribute' } -->` are making this action slow. Removing those or using the default Magento template makes the checkout fast again. However, for some reason the house number and the addition are stored as custom attributes on the billing address, which results in the address looking odd when changing the `details.html` to work as default Magento.

**Screenshots**
N/A

**Please complete the following information**
- Magento version: 2.4.1
- PHP version: 7.3
- TIG PostNL version: 1.9.5

**Comment by tig-jaspersmits** [comment]
Hi @ArjenMiedema ,

Thanks for creating this issue. I've just followed your steps to test this, but for me it doesn't freeze when switching payment method. Are you testing this on a clean Magento installation?

Kind regards,
Jasper
TIG

**Comment by ArjenMiedema** [comment]
@tig-jaspersmits I installed a new Magento Commerce 2.4.1 environment and took the steps I described and I can reproduce it. I use Luma as a theme and have 3 default payment methods active and PostNL from TIG. Also, the issue is not reproduced when logged in, only when checking out as a guest.

The time it freezes is a lot less than on a production shop where we have this issue, but it is still there.

**Comment by tig-jaspersmits** [comment]
Hi @ArjenMiedema ,

I've just setup a clean Magento 2.4.1 environment and installed the latest PostNL version. I've used the test account so that address check fails. Placed a product in my shopping cart and proceed to checkout. I'm checking out as a guest. I saw your video, but this still doesn't happen for me. No freezing occurs. Yes, it loads for a little bit. But that's for recalculating the shipping costs. This is still not reproducible for me.

Kind regards,
Jasper
TIG
